fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraExport.java

```diff
@@ -80,6 +79,10 @@ public Response exportObject(
         final FedoraObjectSerializer serializer =
             serializers.getSerializer(format);
 
+        if (serializer == null) {
+            return status(NOT_FOUND).entity("No " + format + " serialization available for " + resource.getPath())
```

Comment:
I know we're taking `format` as a query param now, but that's wrong. That should just be an `Accept` header.